csaf_2.0/json_schema/csaf_json_schema.json

```diff
@@ -219,9 +219,18 @@
         "distribution": {
           "title": "Rules for sharing document",
           "description": "Describe any constraints on how this document might be shared.",
-          "examples": ["Share only on a need-to-know-basis only.", "Distribute freely.", "Copyright 2019, Example Company, All Rights Reserved."],
-          "type": "string",
-          "minLength": 1
+          "type": "object",
+          "properties": {
+            "tlp": {
+              "type": "string",
+              "enum": ["RED", "AMBER", "GREEN", "WHITE"]
+            },
+            "text": {
```

Comment:
Per our TC Meeting 200527:  Please add reference of intent to subsume FIRST Traffic Light Protocol  https://www.first.org/tlp/ definitions and narrative (presuming this is the consensus).

Reply:
Adding references to the MISP Taxonomies that include consensus JSON taxonomies for key Cyber Domain constructs:

https://github.com/MISP/misp-taxonomies

Specific to TLP and Policies:

https://github.com/MISP/misp-taxonomies/blob/master/iep/machinetag.json
https://github.com/MISP/misp-taxonomies/tree/master/iep2-policy